source/behavior/entities/filters/filters/has_damage.json

```diff
@@ -9,6 +9,35 @@
     "test": { "type": "string", "title": "Test Property", "description": "Returns true when the subject entity receives the named damage type." },
     "operator": { "$ref": "./types/operator.json" },
     "subject": { "$ref": "./types/subject.json" },
-    "value": { "type": "string", "description": "The Damage type to test.", "$ref": "../../../../general/entity/damage_source.json", "title": "Value" }

```

Comment:
If one type is not allowed, then we should check that via the diagnoser I think 🤔

Reply:
You mean flag the damage type that isn't allowed by the diagnoser? Then yeah probably a good idea.